Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
```

Comment:
I think you can actually test the upstream PR prior to it being merged, using something like

```groovy
@Library('github.com/oleg-nenashev/pipeline-library@7b106252928c000a81734f64eaa84f822a680134') _
```

Of course this only matters if [organization scanning](https://ci.jenkins.io/job/Plugins/computation/) is fixed while both PRs are open.